src/merge.c

```diff
@@ -1712,6 +1733,10 @@ int git_merge__iterators(
 	git_vector_foreach(&changes, i, conflict) {
 		int resolved = 0;
 
+		/* Check for merge options in .gitattributes */
+		if ((error = git_merge__lookup_file_favor(repo, conflict->our_entry.path, &opts) < 0))
```

Comment:
Since this modifies `opts` in place, won't the newly set `opts->file_favor` bleed into conflict resolution for other files, processed after the file that had the attribute set?